src/Doctrine/DoctrineHelper.php

```diff
@@ -117,7 +117,7 @@ public function getEntitiesForAutocomplete(): array
         if ($this->isDoctrineInstalled()) {
             $allMetadata = $this->getMetadata();
 
-            /* @var ClassMetadata $metadata */
+            /** @var ClassMetadata $metadata */
```

Comment:
This one looks unnecessary, it hits a variable that doesn't exist

Reply:
This should be `$allMetadata` and adding the second `*` breaks CI https://github.com/symfony/maker-bundle/pull/830/checks?check_run_id=2022138726

Reply:
It applies to the $classname variable not the $allMetadata